### PR TITLE
components: handle enter and return

### LINKED ISSUE #2562: Text input dialog: Enter key on the numeric keyboard doesn't work
image

image

On Windows, nothing happens when you type your password/description and press Enter key on the numeric keyboard.

### PR DESCRIPTION
**Pull request by selsta**
Fixes #2562

### CODE REVIEW

File: components/PasswordDialog.qml [hunk 1]
```diff
@@ -352,7 +353,8 @@ Item {
                     }
                 }
 
-                Keys.onReturnPressed: {
+                onAccepted: {
+                    if (!root.visible) return
```
Comment (xiphon) [inline components/PasswordDialog.qml]: Why do you need this?

Comment (selsta) [reply components/PasswordDialog.qml]: Wanted to avoid something like #1512

Comment (xiphon) [reply components/PasswordDialog.qml]: Would prefer not to do this

Comment (selsta) [reply components/PasswordDialog.qml]: Everywhere or just here?

Comment (xiphon) [reply components/PasswordDialog.qml]: Everywhere. Let me know if this will lead to a bug, would like to reproduce it.

Comment (selsta) [reply components/PasswordDialog.qml]: Yes, it leads to a bug. You can try yourself: Open GUI, go to Transactions -> Payment proof -> Press enter.

**Pull request merged**